gruntwork-install

```diff
@@ -282,8 +288,9 @@ function install_script_module {
         shift
         ;;
       --module-param)
-        local readonly param=$(convert_module_params_format "$2")
-        module_params+=("$param")
+        local -a param=()
+        eval "local -a param=$(convert_module_params_format "$2")"
```

Comment:
Wouldn't this re-declare `param` twice? Also, shouldn't the `eval` be on the data returned from `convert_module_params_format`, rather around the `local -a param=...` declaration?

Reply:
Note my comment above about inlining the `convert_module_params_format` logic here instead. Might make this code a lot cleaner / simpler.

Reply:
It redeclares it, which means that the `local -a param=()` on the line above is vestigial. The reason to do it like this is to control the name of the variable. Otherwise, you're tied to whatever name was used inside `convert_module_params_format`.